src/discounts/components/VoucherSummary/VoucherSummary.tsx

```diff
@@ -163,6 +163,17 @@ const VoucherSummary: React.FC<VoucherSummaryProps> = ({
             <Skeleton />,
           )}
         </Typography>
+        <FormSpacer />
+        <Typography variant="caption">
+          <FormattedMessage
+            id="h65vZI"
+            defaultMessage="Used"
+            description="times voucher used"
+          />
+        </Typography>
+        <Typography>
+          {maybe<React.ReactNode>(() => voucher.used, <Skeleton />)}
```

Comment:
suggestion: We avoid using `maybe`, instead try something like this:
```suggestion
          { voucher?.used ?? <Skeleton /> 
```

Reply:
I have made the suggested change.

Note: Maybe has been used multiple times in the code before. If you look at Voucher Summary file, I made changes to you will find other instances of maybe.

Reply:
We are aware of it - `maybe` was used in the codebase when optional chaining (`?.`) was not yet supported by javascript. We are progressively dropping it.